extensions/twitch/general.py

```diff
@@ -166,19 +166,22 @@ async def count(self, ctx: commands.Context, mbti_type: str) -> None:
 
         await ctx.reply(f"There are {total} {mbti_type} types in the server!")
 
-    @routines.routine(seconds=1)
+    @routines.routine(seconds=60)
     async def midnight(self) -> None:
         channel: twitchio.Channel | None = self.bot.get_channel("timeenjoyed")
         assert channel is not None
 
         for timezone in TIMEZONES:
-            tz = zoneinfo.ZoneInfo(timezone)
-            current = datetime.now(tz)
+            tz: zoneinfo.ZoneInfo = zoneinfo.ZoneInfo(timezone)
+            current: datetime = datetime.now(tz)
+            logger.info(current)
```

Comment:
This is a very verbose amount of logging for info, and non descriptive.

You could make it `debug` and add some information, E.g.
```py
logger.debug("Midnight Routine: It is currently %s in %s", current, timezone)
```

Reply:
verbose meaning wordy? i changed the logger to debug with your description